browser/resources/settings/shortcuts_page/shortcuts_page.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+ /**
+  * Note: This class isn't a Polymer component because it doesn't use any fancy
+  * features - it basically exists to mount our React page inside, and to not
+  * let it be affected by any of the conditional templating upstairs.
+  */
+class ShortcutsPage extends HTMLElement {
+  connectedCallback() {
+    this.attachShadow({ mode: 'open' })
+    import("/commands.bundle.js" as any)
+      .then(() => (window as any).mountCommands(this.shadowRoot))
```

Comment:
Was it not possible to get access to the module's exported functions from the resolved promise? Is that why we're putting `mountCommands` on the global? If it is possible then of course it's preferable to not put it there...

Reply:
Basically, no, not with our current Webpack4 setup. Webpack compiles the module as an IIFE but I'm importing it as an esmodule.

I had a play around with importing it with a script tag, but I was struggling to get the script to load by adding a script tag inside the shadowDOM.

When we migrate to a newer version of Webpack with module output this should be much easier.